prusti-tests/tests/cargo_verify/failing_crate/output.stderr

```diff
@@ -0,0 +1,69 @@
+    Updating crates.io index
+ Downloading crates ...
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+  Downloaded [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
+   Compiling [..]
```

Comment:
The order of `Compiling <dependency name>` is not deterministic because dependencies form a DAG, not a chain. That's why I had to add `[..]` in various places.

Reply:
I see. This is kind of a nitpick then, but I still think it would be cleaner to download and build dependencies separately, then only compare the output of the main run. Unfortunately, this is [not supported by `cargo`](https://github.com/rust-lang/cargo/issues/2644), although that thread has various workarounds. For example, one can run `cargo build` with an "empty" project first, then add the real source code…

Reply:
`[cargo fetch;] cargo build; cargo-prusti` works fine and it's actually useful to test that it builds fine without Prusti; I'll use that. Still, I have no clue how `cargo` manages to switch to the other implementation of `prusti-contracts` without recompiling it when running `cargo-prusti`.